backend/src/main/java/com/praksa/auction/controller/CategoryController.java

```diff
@@ -0,0 +1,28 @@
+package com.praksa.auction.controller;
+
+import com.praksa.auction.model.Category;
+import com.praksa.auction.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@CrossOrigin
```

Comment:
Can we move this cross origin protection into websecurity config or some other config ? So we do not have this annotation on every controller, which we will have more in the future ?